asv.conf.json

```diff
@@ -78,6 +79,19 @@
     //     "six": ["", null],        // test with and without six installed
     //     "pip+emcee": [""],   // emcee is only available for install with pip.
     // },
+    "matrix": {
+        "pyresample": ["1.22.3"],
+        "trollimage": ["1.17.0"],
+        "pyorbital": ["1.7.1"],
+        "pyspectral": ["0.10.6"],
+        "rasterio": ["1.2.10"],
+        "s3fs": [],
+        "h5py": [],
+        "netCDF4": [],
+        "pyhdf": [],
+        "gcsfs": [],
+        "shapely": []
```

Comment:
Thoughts on adding `trollsift`? `dask`? `xarray`? I forget exactly how changing these dependencies resets things. `numpy` could be good too.

Reply:
Good idea. Do you mean as 'latest' or with specific versions?

Reply:
No idea. You said `asv` updates the dependencies marked as "latest" every time the benchmarks are run, right? For dask that would probably mean a new set of benchmarks every couple weeks given their release schedule. So maybe it would be best to specify the version for all of these?

Reply:
What does this dependency matrix do?  It's only listing a single version for each package?  I suppose listing more than one would rapidly make benchmark runs take too long.  What do the empty listings mean?

Reply:
Empty list means fetch the latest version every time benchmarks are run. I can't find the numpy benchmarks now but the one that Martin linked to in the last meeting made it clear that you'd have a separate set of benchmarks in the final HTML for each combination of platform (OS + arch) and version of the dependency in the matrix. So if you had `numpy` and `dask` in the matrix with 2 versions each, you'd get 4 total runs of the benchmarks for each numpy version with each dask version...I think.

Reply:
I added trollsift, dask, xarray and numpy to the matrix. Any more comments?